test/tests/functional/pbs_reliable_job_startup.py

```diff
@@ -1483,10 +1489,10 @@ def test_t1(self):
 %s
 """ % (jid, self.momA.hostname, self.momD.hostname, self.momE.hostname,
             self.fib37_value, self.fib37_value, self.fib37_value,
-            self.momA.hostname, self.momD.hostname, self.momE.hostname,
-            self.momA.hostname, self.momA.hostname, self.momA.hostname,
-            self.momD.hostname, self.momD.hostname, self.momD.hostname,
-            self.momE.hostname, self.momE.hostname, self.momE.hostname)
+            self.momA.shortname, self.momD.shortname, self.momE.shortname,
+            self.momA.hostname, self.momA.hostname, self.momA.shortname,
+            self.momD.hostname, self.momD.hostname, self.momD.shortname,
+            self.momE.hostname, self.momE.hostname, self.momE.shortname)
```

Comment:
Why have some of these remained hostname and others changed to shortname? Seems like we would want to use shortname everywhere to avoid hostname mismatches.

Reply:
The short hostnames match the "hostname" output while the full hostnames match the output when validating the entries in $PBS_NODEFILE as  done in the job script as:

for h in `cat $PBS_NODEFILE` do
     echo "HOST=$h"
     ...
done

Reply:
Works for me. You still have my signoff.